maven-xml-impl/pom.xml

```diff
@@ -26,8 +26,8 @@ under the License.
   </parent>
 
   <artifactId>maven-xml-impl</artifactId>
-
-  <name>Maven XML Implementation</name>
+  <name>Implementation of Maven API XML</name>
```

Comment:
see above

Reply:
This module actually contains the implementation of the XML `Dom` interface defined by the API.